build-system/npm-publish/test/get-extensions.test.js

```diff
@@ -0,0 +1,8 @@
+const test = require('ava');
+const {getExtensionsAndComponents} = require('../utils');
+
+test('get-extensions', (t) => {
+  const extensions = getExtensionsAndComponents();
+  const names = extensions.map(({extension}) => extension);
+  t.true(names.includes('bento-accordion'));
```

Comment:
can you also test that it includes an "extension" as well? I think components = `src/bento/components` and extensions = `extensions/`

Reply:
I actually don't want to check that here. All the extensions will be ported over soon which would cause this test to fail.